reference-architectures/deploy-arch-ibm-vsi-quickstart.md

```diff
@@ -0,0 +1,148 @@
+---
+
+copyright:
+  years: 2025
+lastupdated: "2025-09-26"
+
+keywords: vsi, virtual server instance, vpc, landing zone, deployable architecture, terraform, quickstart
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: "Jordan Williams"
+
+# The release that the reference architecture describes
+version: 5.8.1
+
+# Whether the reference architecture is published to Cloud Docs production.
+# When set to false, the file is available only in staging. Default is false.
+production: false
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vsi-ef663980-4c71-4fac-af4f-4a510a9bcf68-global
+
+docs: https://cloud.ibm.com/docs/secure-infrastructure-vpc
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/blob/main/reference-architectures/vsi-qs.svg
+
+related_links:
+  - title: "Landing zone for applications with virtual servers - Quickstart - Basic and simple"
+    url: "https://cloud.ibm.com/docs/deployable-reference-architectures?topic=deployable-reference-architectures-vsi-ra-qs"
```

Comment:
this is pointing to the wrong quickstart - again update line 46 and use unique name